db/projects/litecoin/project.yaml

```diff
@@ -1,15 +1,46 @@
-name: litecoin
+name: Litecoin
+start_date: '2011–10-07’
+web: https://www.litecoin.org/
 assets:
-  - id: ltc
-    type: blockchain
-    name: Litecoin
-    symbol: LTC
-    networks:
-      - id: main
-        type: main
-        name: mainnet
-        proof_type: PoW
-        mineable: true
-    webids:
-      coinmarketcap: litecoin
-      cryptocompare: ltc
+- id: ltc
+  symbol: LTC
+  name: Litecoin
+  type: blockchain
+  networks:
+  - id: main
+    type: main
+    name: mainnet
+    proof_type: PoW
+    algorithm: Scrypt
+    mineable: true
+    target_block_time: 150
+    tools:
+    - type: explorer
+      url: http://explorer.litecoin.net/chain/Litecoin
+    - type: explorer
+      url: https://chainz.cryptoid.info/ltc/
+  max_supply: 84000000
+  first_distribution:
+  - title: Premined
+    amount: 150
+  denominations:
+    plural: litecoins
+    symbol: "Ł"
+    subunits:
+    - unit: 0.001
+      name: lites
+    - unit: 1.0e-6
+      name: photons
+    - unit: 1.0e-8
+      name: litoshis
+team:
+- name: Charlie Lee
+  role: founder
+contacts:
+  email: contact@litecoin.org
+tags:
+- technology: segwit
```

Comment:
tag musi byt dohromady a v uvozovkach, pokud je prefixovanej (technology:), to je kvuli yamlu jinak to bere jinak
```
tags:
- "technology:segwit"
```